src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Tests;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.Tests
+{
+    public class HttpClientTest
+    {
+        [Fact]
+        [OuterLoop]
+        public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
+        {
+            // TODO: This server path will change once the final test infrastructure is in place (Issue #1477).
+            const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
+            
+            var client = new HttpClient();
```

Comment:
I suppose we get explicitly call `Dispose()` on it prior to exiting the test method.  However, most (if not all) of the other test collateral we have doesn't do that right now unless we are explicitly testing "dispose" logic.

The important guideline in writing the test methods is to not exit them with outstanding async operations running.  And this method does ensure that the async operation is cleanly done.

But I'll add the `Dispose()' here anyways.